src/dmd/globals.d

```diff
@@ -462,13 +462,45 @@ nothrow:
         return buf.extractString();
     }
 
+    /* Checks for equivalence,
+     * a) comparing the filename contents (not the pointer), case-
+     *    insensitively on Windows, and
```

Comment:
FYI, macOS also has a case insensitive filesystem by default.

Reply:
Ah, interesting. The `FileName.equals()` function used here only has a Windows special case.